### DISCUSSION TITLE
server url address for window.ai?

### DISCUSSION THREAD

**Discussion by SAWSYSTEM** [louisgv/local.ai#83] [Q&A]
sorry if this question comes off as stupid, new to all of this. i understand the port and concurrency numbers relate to the url but i don't understand what order they come in or if it's even localhost or not

**Comment by louisgv** [comment]
@SAWSYSTEM see https://github.com/louisgv/local.ai/wiki - it should just work out of the box with window.ai, you don't really need to configure much iirc :d... (point it to localhost:8000)
